experimental/traffic-portal/src/app/core/profiles/profile-detail/profile-detail.component.html

```diff
@@ -0,0 +1,61 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<mat-card>
+    <tp-loading *ngIf="loading"></tp-loading>
+    <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="profile">
+        <mat-card-content>
+            <mat-form-field *ngIf="!new">
+				<mat-label>ID</mat-label>
+				<input matInput type="text" name="id" disabled readonly [defaultValue]="profile['id']" />
+			</mat-form-field>
+            <mat-form-field>
+                <mat-label>Name</mat-label>
+                <input matInput type="text" name="name" [(ngModel)]="profile['name']" [ngModelOptions]="{standalone: true}" required />
+            </mat-form-field>
+            <mat-form-field>
+                <mat-label>CDN</mat-label>
+                <mat-select name="cdn" [(ngModel)]="profile['cdn']" [ngModelOptions]="{standalone: true}" required>
```

Comment:
When creating a new profile, if this field is left blank, pressing the submit button will send the request instead of focusing this field and requesting input (works currently for name and description)